include/net/net_if.h

```diff
@@ -1266,9 +1269,50 @@ void net_if_ipv6_dad_failed(struct net_if *iface, const struct in6_addr *addr);
  * @return Pointer to IPv6 address, NULL if not found.
  */
 struct in6_addr *net_if_ipv6_get_global_addr(struct net_if **iface);
+
 #else
-#define net_if_ipv6_select_src_addr(...)
+#define net_if_config_ipv6_get(...) -EOPNOTSUPP
```

Comment:
These defines do not seem to belong to this patch called "net_if: Define void net_if.h functions for CONFIG_NET_OFFLOAD"

Reply:
These function pop out since the preprocessor does not anymore remove the code, but instead the compiler sees it now with IS_ENABLED() being used.